gcloud/orb.yaml

```diff
@@ -359,12 +364,13 @@ jobs:
           steps:
             - run: |
                 gcloud functions deploy <<parameters.funcname>> --memory=<<parameters.memory>> \
```

Comment:
Think it's worth deprecating/removing the `memory` flag at this point? eg. just force everyone to use `args`?

Reply:
That could be doable. My rationale for leaving it was that it is one of those fields that has a specific set of allowable memory values, but you'd still have to either read the orb description first anyway. Might as well have users only reference one place: the `gcloud` helptext.